webrtc.html

```diff
@@ -1114,8 +1115,8 @@
             <p>Append <var>operation</var> to [[<a>operations</a>]].</p>
           </li>
           <li>
-            <p>If the length of [[<a>operations</a>]] is exactly 1, execute
-            <var>operation</var>.</p>
+            <p>If the length of [[<a>operations</a>]] is exactly 1, queue a task
```

Comment:
It's not OK to queue here, because each operation has a synchronous part that needs to run synchronously in the typical case, to ensure errors are thrown correctly.

The reason this matters is because we follow a pattern of doing all input validation synchronously, such that a JavaScript developer can observe the that the returned promise is in rejected state on return when observed in a JavaScript debugger.